demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -28,7 +28,7 @@ def dump(self, dir: Path, marketplace: MarketplaceVersions) -> None:
         dir.mkdir(exist_ok=True, parents=True)
         try:
             IntegrationScriptUnifier(
-                input=str(self.path.parent), output=str(dir), marketplace=marketplace
+                input=str(self.path.parent), output=str(dir), marketplace=marketplace, force=True
```

Comment:
Why do we need to use force?

Reply:
That's the way we currently use it in the original `create-content-artifacts`, I'll take a look what difference this makes